xtext/editorserver/model2plantuml.egl

```diff
@@ -89,12 +65,12 @@ operation Any getReferenced() {
 }
 operation getReferenceLabel(s, t, r) {
 	
-	for (ref in s.eClass().getEAllReferences()) {
-		if (ref.eType.isSuperTypeOf(t.eClass()) and r <> ref) {
+//	for (ref in s.eClass().getEAllReferences()) {
+//		if (ref.eType.isSuperTypeOf(t.eClass()) and r <> ref) {
 			return " : " + r.name;
-		}
-	}
-	return "";
+//		}
+//	}
+//	return "";
```

Comment:
Is this commented out for a reason? should remove or add an comment to explain why you might want un-comment it

Reply:
I have added an explanatory comment.